### DISCUSSION TITLE
Welcome to Ecole Discussions!

### DISCUSSION THREAD

**Discussion by AntoinePrv** [ds4dm/ecole#107] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Help us determine what learning task, reward function, and benchmarking instance set are worth investigating by the community.
  * Ask questions for help on using Ecole.
  * Share ideas on how to use Ecole and direction to improve the API.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hanmanming** [comment]
how can I  install ecole?why i can't install it by pip or conda,do i need to install it by WSL??
